lib/commons/reuasabel_bottom_button.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+import 'package:furniture/commons/reusable_button.dart';
+
+class ReusableBottomButton extends StatelessWidget {
+  const ReusableBottomButton({super.key, required this.bottomButton});
+  final ReusableButton bottomButton;
+  @override
+  Widget build(BuildContext context) {
+    Size size = MediaQuery.of(context).size;
+    return BottomAppBar(
+        elevation: 0.0,
+        child: Row(
+          children: [
+            Expanded(
+              child: Padding(
+                padding: const EdgeInsets.all(20.0),
+                child: SizedBox(
+                    height: 70, width: size.width, child: bottomButton),
+              ),
+            ),
+          ],
```

Comment:
As row represent multi-child widget but in this code we only have one widget in row. 
We can remove Row.